cmd/boulder-va/main.go

```diff
@@ -89,6 +90,13 @@ func main() {
 
 	sbc := newGoogleSafeBrowsing(c.VA.GoogleSafeBrowsing)
```

Comment:
Fixed in https://github.com/letsencrypt/boulder/pull/2446/commits/d1f8c3d2e013645e6992298bea706dd27fdfbd30